### ISSUE TITLE
include the function parse_rpu_file into the c-api?

### ISSUE THREAD

**Issue by erazortt** [quietvoid/dovi_tool#164]
I would like to use the functionality parse_rpu_file in the dovi library created from the dolby_vision directory.
Would that be possible?

**Comment by quietvoid** [comment]
Sure, it's totally possible.
It would probably take a char* of the file path, and return an allocated list of the RPUs, which may be large, though.

**Comment by erazortt** [comment]
Yes, that would be exactly what I thought it to be like and what I’d need!

**Comment by quietvoid** [comment]
I've implemented as a list of the existing `DoviRpuOpaque`.
You can find a usage example here: https://github.com/quietvoid/dovi_tool/blob/remove_utilities/dolby_vision/examples/capi_rpu_file.c

Does this work for you?

**Comment by erazortt** [comment]
Yeah, works for me. I will continue to test with some real life rpu files, but its looking good so far. Thanks a lot, Cheers!
